src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -166,6 +167,8 @@ def setUp(self):
         test_clients = self.params.get("test_clients", "/run/hosts/*")
         server_count = self.params.get("server_count", "/run/hosts/*")
         client_count = self.params.get("client_count", "/run/hosts/*")
+        self.nvme_parameter = self.params.get("bdev_class", '/server_config/',
+                                              None)
```

Comment:
If it helps make this fit in one line, `None` is the default and not required to be be explicitly stated.